src/System.Transactions.Local/src/System/Transactions/TransactionException.cs

```diff
@@ -112,25 +112,25 @@ internal static TransactionException Create(string message, Guid distributedTxId
             return new TransactionException(message);
         }
 
-        internal static TransactionException Create(string message, Exception innerException, Guid distributedTxId)
+        internal static TransactionException Create(string? message, Exception? innerException, Guid distributedTxId)
         {
-            string messagewithTxId = message;
+            string? messagewithTxId = message;
             if (IncludeDistributedTxId(distributedTxId))
                 messagewithTxId = SR.Format(SR.DistributedTxIDInTransactionException, messagewithTxId, distributedTxId);
 
             return Create(messagewithTxId, innerException);
         }
 
-        internal static TransactionException Create(TraceSourceType traceSource, string message, Exception innerException, Guid distributedTxId)
+        internal static TransactionException Create(TraceSourceType traceSource, string? message, Exception innerException, Guid distributedTxId)
```

Comment:
`innerException` should be nullable right?

Reply:
yes